src/ecalc/libraries/libecalc/common/tests/integration/snapshots/test_all_energy_usage_models/test_all_results/all_energy_usage_models_v3.json

```diff
@@ -1044,9 +1044,9 @@
                     "typ": "STREAM_DAY",
                     "unit": "MW",
                     "values": [
-                        644.62915,
```

Comment:
here the values decrease...do you know why? was that expected?

Reply:
as far as I can see they have all 1 MW in PAC

Reply:
This is the power capacity margin, which should decrease when the power increase.